improver/utilities/mathematical_operations.py

```diff
@@ -200,68 +230,64 @@ def perform_integration(
             lower_bounds_cube (iris.cube.Cube):
                 Cube containing the lower bounds to be used during the
                 integration.
-            integrated_cube (iris.cube.Cube):
-                Cube that will be used for storing the output of the
-                integration containing the most appropriate coordinates.
 
         Returns:
             iris.cube.Cube:
                 Cube containing the output from the integration.
 
         """
-        # Create a zip for looping over.
-        levels_tuple = list(zip(
+        def skip_slice(upper_bound, lower_bound, direction,
+                       start_point, end_point):
+            """Conditions under which a slice should not be included in
+            the integrated total"""
```

Comment:
I think it would be useful here to state that all inputs are single float values, even if you don't declare each one separately.